example/gamemode.ini

```diff
@@ -43,9 +43,7 @@ inhibit_screensaver=1
 ; Setting this to the keyphrase "accept-responsibility" will allow gamemode to apply GPU optimisations such as overclocks
 ;apply_gpu_optimisations=0
 
-; You must set these to tell gamemode the Vendor of your graphics card, as well it's device number on the system (usually 0)
-; Vendor must be one of 0x10de (NVIDIA), 0x1002 (AMD) or 0x8086 (Intel)
-;gpu_vendor=0x0000
+; The device number on the system (usually 0)
```

Comment:
Still think this is a bit unclear - "DRM device number" would be better.

Reply:
Hmm yeah, I guess I could also put an example with the dev path

Reply:
bfe0903 tries to make this better :)

Reply:
Thanks, merged.